docs/station.md

```diff
@@ -0,0 +1,59 @@
+# station (Private Beta)
+
+A hybrid station has the capability to scan a resource within your Virtual Private Cloud (VPC) environment based on the inputs provided through the HCP UI.
+The `station` command is used to fetch scan jobs from [HCP](https://portal.cloud.hashicorp.com/) and execute them in your VPC. It will scan any available jobs, otherwise it will sleep and wait for a job to be queued in HCP.
+
+## Setting Up
+
+In order to run `station` in your environment there are 3 configuration steps required in HCP and your local environment.
+
+### Create a Service Principal
+
+You will need to log into the HCP web portal to create your Service Principals. You can follow the HCP docs to create your [Project Service Principals](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/service-principals#project-level-service-principals-1). You will need to create the Service Principals at Admin level, more information about this level contained in [these](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/users#project-role) HCP docs. You will also need to generate a [service principal key](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/service-principals#generate-a-service-principal-key) that gives you a client id and client secret that you will use in the next few steps.
+
+### Create a Station in the HCP Portal
+
+> **_Note:_** Before you create a station, there is currently no way to delete stations via the UI
+
+Now, you will need to initiate a station in the HCP web portal. If it's your first time using `Vault Radar` in HCP, you will need to configure a data source of your choice in order to get the `Station` tab view. This data source will not be used for running the station at all. This is a known UX issue that will be fixed in a future iteration.
+
+Once you have the `Station` tab view, you can start working through the station creation flow. This will give you a Station ID you will use in your local environment.
+
+### Configure your Local Environment
+
+To run station you will need your client id and client secret from the service principal you just created, as well as your project and station id, which will be provided to you during your station creation flow.
+
+To run station, the following environment variables must be set:
+
+```bash
+export HCP_PROJECT_ID=<HCP_PROJECT_ID>
+export HCP_CLIENT_ID=<HCP_CLIENT_ID>
+export HCP_CLIENT_SECRET=<HCP_CLIENT_SECRET>
+
+# The Station Id that was provided after creation
+export HCP_RADAR_STATION_ID=<STATION_ID>
```

Comment:
I think we also should add needing a github token here?

Reply:
Can probably reference this: https://github.com/hashicorp-guides/vault-radar-demo/blob/main/docs/git.md#authentication

Reply:
Yeah! good call